### PR TITLE
ADDING NEW FEATURE COMPMNENT

### PR DESCRIPTION
**Pull request by mrmayank2710**
hey there im adding new feature component

### CODE REVIEW

File: components/feature-task.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<div class="bg-black relative min-h-screen min-w-screen">
+    <div class="relative lg:absolute left-0 top-0 w-full lg:w-1/2 h-64 lg:h-full">
+        <img src="C:\Users\Mayank\Desktop\PYTHON PRO\mockvita\tails\README.md\Vice-City-1920x1080.png"
```
Comment (thinkverse) [inline components/feature-task.html:3]: The use of copyrighted trademarks isn't a good idea, better to use something like [Unsplash]. Also, the path is absolute based on your system. If you want to include an image, browse around Unsplash, they have a great selection of free images to use. 👍

[unsplash]: https://unsplash.com/

File: components/feature-task.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div class="bg-black relative min-h-screen min-w-screen">
```
Comment (thinkverse) [inline components/feature-task.html:1]: The pure black is to harch for background colors, which paired with the color choice for the `h2` on line 11, results in horrible contrast not even hitting the [WCAG A] standard.

[wcag a]: https://webaim.org/resources/contrastchecker/

File: components/feature-task.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            class="lg:absolute inset-0 w-full h-full object-cover">
+    </div>
+    <div class="max-w-6xl mx-auto flex items-center pb-20 pt-0 lg:pt-24 flex items-center justify-center h-full">
+        <div class="w-1/2 hidden lg:block"></div>
+        <div
+            class="relative w-full lg:w-1/2 h-full pl-12 pr-10 lg:pl-16 lg:pr-0 flex flex-col items-start justify-center">
+            <p class="text-sm uppercase font-bold text-indigo-500 tracking-wide">Cool Features</p>
+            <h2 class="text-4xl font-bold text-gray-900 mt-5 leading-tight">RAZER VICE CITY <br> FOR GAMERS BY GAMERS.
+            </h2>
+            <p class="text-gray-600 text-base mt-5">Are you ready for GAMING TOURNAMENTS!!!!!!!!!!!!!!!!!!!!!!!!!!!</p>
```
Comment (thinkverse) [inline components/feature-task.html:13]: To many exclamation marks. 🙂
```suggestion
            <p class="text-gray-600 text-base mt-5">Are you ready for GAMING TOURNAMENTS!</p>
```

**Review by thinkverse — CHANGES_REQUESTED** [review]

**Comment by tnylea** [comment]
@mrmayank2710 Yeah, if you could replace this image with an image from unspash and give it an absolute URL, then I can see about getting this one merged in.

I may do a few touch-ups as well, just so that way it doesn't look like the other featured blocks :)

Thanks!

File: components/feature-task.html (hunk 2 above)
Comment (mrmayank2710) [reply components/feature-task.html:1]: okay i will change the background color.

File: components/feature-task.html (hunk 3 above)
Comment (mrmayank2710) [reply components/feature-task.html:13]: im will remove them too ! sry!

**Pull request closed without merging**